docs/getting_started/setup_slack.md

```diff
@@ -0,0 +1,118 @@
+# Setup Slack Events API receiver
+
+This document walks you through how to set up Slack as an input source for your live blog and use their `Events API` to receive new updates.
+
+## Create a Slack app
+
+First, you will need to create [a Slack app](https://api.slack.com/apps/new).
+You may be prompted to create it from scratch or from an App Manifest. For the purpose of this tutorial, you can create your app from scratch.
+
+Fill out your app name and pick a workspace to develop your app in. Click the `Create App` button and you should see your app's **Basic Information** page.
+
+## Retrieve your signing secret
+
+Slack signs all the requests it sends using a secret key.
+
+Wagtail Live needs that key to confirm that requests sent to the `wagtail_live/slack/events` endpoint  comes from Slack by verifying their unique signature. (You will learn more about this endpoint later)
+
+To find your secret key, go back to your app's page and navigate to the **Basic Information** section on the left sidebar. 
+
+Scroll down to **App Credentials** and you'll find your Signing Secret in the `SLACK_SIGNING_SECRET` field.
+
+Grab it and keep it in a safe place.
+
+### Save your signing secret
+
+Save the signing secret as an environment variable like this:
+
+**On Windows** (cmd.exe):
+
+```doscon
+> SET SLACK_SIGNING_SECRET=<your-signing-secret>
+```
+
+**On GNU/Linux or MacOS** (bash):
+
+```console
+$ export SLACK_SIGNING_SECRET=<your-signing-secret>
+```
+
+### Add your signing secret to `settings`
+
+Add the following in your `settings.base` file:
+```python
+    SLACK_SIGNING_SECRET = os.environ['SLACK_SIGNING_SECRET']
+```
+
+## Receiving Slack events
+
+In order to receive events like when a message is sent in a channel, you need to hook a **Request URL** that Slack uses to send HTTP POST requests corresponding to events you specify. This URL needs to be publicly accessible. 
+
+In the development phase, you will need a development proxy that creates a public URL and tunnels requests to your local server when you run `python3 manage.py runserver` in order to receive events from Slack.
+
+If you are comfortable with ngrok, start ngrok on port 8000 otherwise you can see how to [Set up a local web server with ngrok](setup_ngrok.md).
```

Comment:
```suggestion
If you are comfortable with ngrok, start ngrok on port 8000. Otherwise you can see how to [Set up a local web server with ngrok](setup_ngrok.md).
```